CHANGELOG.md

```diff
@@ -9,6 +9,12 @@ See [GitHub releases](https://github.com/mll-lab/php-utils/releases).
 
 ## Unreleased
 
+## v3.1.0
+
+### Changed
+
+- Add optional `Project`-column to Illumina Sample Sheets (V2) for NovaSeq
```

Comment:
```suggestion
- Add optional column `Project` to Illumina Sample Sheets (V2) for NovaSeq
```

The former makes it sound like *a column of type Project* or something similarly vague, the latter makes it clear that it refers to *a column named Project*.